### PR TITLE
Add vollseg-napari recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

---

Depends on:
- https://github.com/conda-forge/staged-recipes/pull/17844

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be automatically open, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the @conda-forge/napari team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@kapoorlab, do you agree with being added as a maintainer?

**Comment by goanpeca** [comment]
Depends on:
- https://github.com/conda-forge/staged-recipes/pull/17844

**Comment by goanpeca** [comment]
Pinging @kapoorlab. Do you agree with being added as a maintainer :) ?

**Comment by kapoorlab** [comment]
@goanpeca yes I agree being added as a maintainer for this one :)

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

**Comment by kapoorlab** [comment]
@goanpeca many thanks for doing this.

File: recipes/vollseg-napari/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+about:
+  home: https://github.com/kapoorlab/vollseg-napari
+  summary: Irregular cell shape segmentation using VollSeg
+  license: BSD-3-Clause
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - goanpeca
+    - jaimergp
```
Comment (carterbox) [inline recipes/vollseg-napari/meta.yaml:46]: Not all the maintainers have commented!

File: recipes/vollseg-napari/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/vollseg-napari-{{ version }}.tar.gz
+  sha256: bc1a435f3fbd08a38f5a6fae0b7d7581ee91e56e46f2ae933f59f1d0de8fe57f
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
Comment (carterbox) [inline recipes/vollseg-napari/meta.yaml:15]: Please convert this recipe to `noarch: python` or explain why it doesn't qualify. (missing dependencies on a platform (Windows) are fine for noarch: python).

File: recipes/vollseg-napari/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/vollseg-napari-{{ version }}.tar.gz
+  sha256: bc1a435f3fbd08a38f5a6fae0b7d7581ee91e56e46f2ae933f59f1d0de8fe57f
+
+build:
+  skip: true  # [py<37]
```
Comment (carterbox) [inline recipes/vollseg-napari/meta.yaml:13]: Upstream README says there is a maximum python version? <3.9?

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/vollseg-napari/meta.yaml (hunk 1 above)
Comment (jaimergp) [reply recipes/vollseg-napari/meta.yaml:46]: I give my consent :)

File: recipes/vollseg-napari/meta.yaml (hunk 3 above)
Comment (jaimergp) [reply recipes/vollseg-napari/meta.yaml]: setup.py says >=3.7 🤷

**Comment by kapoorlab** [comment]
> @kapoorlab any chance you can cut a new release on PyPI or tag on Github? We need [this commit](https://github.com/Kapoorlabs-CAPED/vollseg-napari/commit/d303207c03278448608e04b8dc711443ed1459fc) to fix the issues with CI.

Hello, yes I pushed a new release to pypi and also a new [tag](https://github.com/Kapoorlabs-CAPED/vollseg-napari/releases/tag/Condaforge).

File: recipes/vollseg-napari/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
+{% set name = "vollseg-napari" %}
+{% set version = "2.3.4" %}
```
Comment (jaimergp) [inline recipes/vollseg-napari/meta.yaml:2]:
```suggestion
{% set version = "2.3.5" %}
```

File: recipes/vollseg-napari/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/vollseg-napari-{{ version }}.tar.gz
+  sha256: bc1a435f3fbd08a38f5a6fae0b7d7581ee91e56e46f2ae933f59f1d0de8fe57f
```
Comment (jaimergp) [inline recipes/vollseg-napari/meta.yaml:10]:
```suggestion
  sha256: b5176c83e256b4191b3d3ca30e0e3d8fc860983982368e97e623e193dabf0851
```

**Comment by jaimergp** [comment]
@carterbox I think this is ready for re-review :)

**Pull request merged**